app-misc/pwsafe/pwsafe-0.2.0-r5.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A Password Safe compatible command-line password manager"
+HOMEPAGE="http://nsd.dyndns.org/pwsafe/"
+SRC_URI="http://nsd.dyndns.org/pwsafe/releases/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="X libressl readline"
+
+DEPEND="sys-libs/ncurses:0=
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	readline? ( sys-libs/readline:0= )
+	X? ( x11-libs/libSM
+		x11-libs/libICE
+		x11-libs/libXmu
+		x11-libs/libX11 )"
+RDEPEND="${DEPEND}
+	!app-admin/passwordsafe"
+
+src_prepare() {
+	eapply -p0 "${FILESDIR}/${P}-cvs-1.57.patch"
+	eapply -p0 "${FILESDIR}/${P}-printf.patch"
+	eapply -p0 "${FILESDIR}/${P}-fake-readline.patch"
+	eapply -p0 "${FILESDIR}/${P}-man-page-option-syntax.patch"
+	eapply -p0 "${FILESDIR}/${P}-XChangeProperty.patch"
+	eapply_user
+}
+
+src_configure() {
+	econf $(use_with X x) $(use_with readline)
```

Comment:
```suggestion
	econf \
	    $(use_with X x) \
	    $(use_with readline)
```

Reply:
I thought it was a fairly short line and totally readable as is, but as you don’t like it I’ve changed it.